packages/rspack-dev-server/tests/__snapshots__/normalizeOptions.test.ts.snap

```diff
@@ -128,9 +127,8 @@ exports[`normalize options snapshot react-refresh client added when react/refres
 {
   "main": [
     "<prefix>/rspack-dev-client/dist/reactRefresh.js",
-    "<prefix>/rspack-dev-client/dist/devServer.js",
-    "<prefix>/dist/clients/WebSocketClient.js",
```

Comment:
Is this expected?

Reply:
yes, because it was added to the `provider`